Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -891,6 +891,18 @@ lemma ChartedSpace.sum_chartAt_inr (x' : M') :
     chartAt H (Sum.inr x') = (chartAt H x').lift_openEmbedding (X' := M ⊕ M') IsOpenEmbedding.inr :=
   rfl
 
+@[simp]
```

Comment:
I'd appreciate another opinion whether marking these lemmas simp is fine. I'm going purely by analogy: most `PartialHomeomorph.something_apply` lemmas are simp.